frontend/src/components/Table/Table.tsx

```diff
@@ -119,6 +119,8 @@ export interface TableProps<T extends object = {}> extends TableOptions<T> {
   filterable?: boolean;
   filter?: { [key in keyof T]?: any };
   onFilterChange?: (values: any) => void;
+  /** have page selection menu drop-up to avoid contianer growing in some scenraios */
+  dropUp?: boolean;
```

Comment:
nit picky but "contianer", "scenraios"

Reply:
Lol nit picky is good, rather not have a million typos from myself :)